cmake/modules/dts.cmake

```diff
@@ -233,6 +238,18 @@ if(SUPPORTS_DTS)
     message(STATUS "Generated device_extern.h: ${DEVICE_EXTERN_H}")
   endif()
 
+
+  execute_process(
+    COMMAND ${PYTHON_EXECUTABLE} ${GEN_DRIVER_KCONFIG_SCRIPT}
+    --kconfig-out ${DTS_KCONFIG}
+    --bindings-dirs ${DTS_ROOT_BINDINGS}
+    WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
+    RESULT_VARIABLE ret
+    )
```

Comment:
Maybe add a status message? Configure step is already cluttered enough, though...

Reply:
Yeah, don't know what the status message would say/add and how much anyone pays any attention to them.